schemas/session-replay/_common-wireframe-schema.json

```diff
@@ -0,0 +1,42 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "_common-wireframe-schema.json",
+  "title": "CommonWireframe",
+  "type": "object",
+  "description": "Schema of common properties for Wireframe events type.",
+  "required": [
+    "id",
+    "x",
+    "y",
+    "width",
+    "height",
+    "type"
```

Comment:
it is missing in the `properties` below. Should it be in the `required` list here? Seems it comes only with `ShapeWireframe`.

Reply:
not it is a left over from a prev iteration ;)